tests/pytest.ini

```diff
@@ -2,6 +2,7 @@
 addopts = -s -v
 markers =
     sanity: marks tests as sanity
+    sonic: mark tests with sonic which need sonic dut
```

Comment:
just call it dut (don't name any specific vendor)

Reply:
added sonic, as we will have multiple use cases from multiple customers and we can run based on mark tag if we get to test on specific DUT/customer.